### ISSUE TITLE
CircleCI MySQL builds are broken

### ISSUE THREAD

**Issue by kennyadsl** [solidusio/solidus#3271]
We have all the CircleCI MySQL builds broken. 

Here's an example: https://circleci.com/gh/solidusio/solidus/12551

I think this could be due to [recent updates of the CircleCI docker images](https://hub.docker.com/r/circleci/mysql/tags), here's the error:

```
LoadError:
  Error loading the 'mysql2' Active Record adapter. Missing a gem it depends on? libmariadbclient.so.18: cannot open shared object file: No such file or directory - /home/circleci/solidus/vendor/bundle/ruby/2.5.0/gems/mysql2-0.5.2/lib/mysql2/mysql2.so
```

**Comment by kennyadsl** [comment]
Before each specs run our CircleCI config retrieves the cached gems saved into a folder shared between all builds. Gems cached include native extensions that are needed for some gems to work on the OS used (eg. nokogiri, mysql). At the moment I think that the recent mysql docker image update changed some path or library used by our cached native extension.

The only thing that we can do is clear the cache (by bumping the key used) and regenerate native extension for that specific mysql gem version using the new docker image released.

I'm doing some experiments [here](https://circleci.com/gh/nebulab/solidus/tree/kennyadsl%2Ftest-circle-no-cache) and if that works I'll submit a PR soon.

**Comment by kennyadsl** [comment]
Closed with #3272
